open-craft/edx-platform
# FAL-984 port recent xss patches to common juniper branch

**Sandbox**:



-    LMS: https://fal-984-xss-fixes.opencraft.hosting/
-    Studio: https://studio.fal-984-xss-fixes.opencraft.hosting/


**Test instructions**:

- verify that the changes match the patches (these all applied cleanly):
  - https://discuss.openedx.org/t/security-im-patch-for-xss-fixes-7/4198
  - https://discuss.openedx.org/t/security-im-patch-for-xss-fixes-8/4231
  - https://discuss.openedx.org/t/security-im-patch-for-xss-fixes-9/4297
- verify that the notes xss fix is not required for this branch: https://github.com/edx/edx-platform/commit/f1324710af5559375e0e13cb0d58f3b820c9ae27 | https://discuss.openedx.org/t/security-patch-for-xss-fixes-on-learner-notes-feature/4241
- verify that an instance with this branch provisions successfully (check the [sandbox](https://manage.opencraft.com/instance/24700/))
- browse through the files changed and run a quick check of the associated functionality on the sandbox

**Author notes**:

- I directly applied and committed the patches because the patches were applied in many small commits upstream (according to the ticket), so this is cleaner.

**Reviewers**:

(as firefighters)
